rcl_lifecycle/src/default_state_machine.c

```diff
@@ -659,6 +660,9 @@ rcl_lifecycle_init_default_state_machine(
   rcl_ret_t ret = RCL_RET_ERROR;
   // Used for concatenating error messages in the fail: block.
   const char * fail_error_message = "";
+  char * init_error_string = "";
+  char * fini_error_string = "";
+  rcl_allocator_t default_allocator;
```

Comment:
Why not initializing it directly? That would say it later on.

```suggestion
  rcl_allocator_t default_allocator = rcl_get_default_allocator();
```

Reply:
`default_allocator` is only used in failure case.  
I think it happens seldom.  
So need not call `rcl_get_default_allocator()` each time.  
What do you think ?